shared/js/media/media_utils.js

```diff
@@ -31,14 +31,14 @@ var MediaUtils = {
     }
     var sizeDecimal = i < 2 ? Math.round(size) : Math.round(size * 10) / 10;
 
-    return navigator.mozL10n.formatValue('byteUnit-' + units[i]).then(
+    return document.l10n.formatValue('byteUnit-' + units[i]).then(
```

Comment:
both, l10n.js and l20n.js have proxies to each other because of this problem. So any app with `l10n.js` will work with `document.l10n` and any app using `l20n.js` will handle `navigator.mozL10n` :)